CMakeLists.txt

```diff
@@ -121,13 +121,27 @@ add_compile_definitions(
     QT_USE_QSTRINGBUILDER
 )
 
+if(BUILD_WASM)
```

Comment:
could add something to the CHANGELOG for this :-)

Reply:
done :D